tests/testAutopas/tests/utils/ConstexprMathTest.cpp

```diff
@@ -0,0 +1,53 @@
+/**
+ * @file ConstexprMathTest.cpp
```

Comment:
It would be really cool if these tests could also check whether the functions are really evaluated `constexpr`, however, I'm not entirely sure how you could do that.

Reply:
Relevant discussion: https://stackoverflow.com/questions/37991911/how-to-detect-if-a-func-is-a-constexpr-and-mark-other-func-constexpr-dependin

So probably check at compile time by assigning the results to a `constexpr` variable?

Reply:
done in [4a5aece](https://github.com/AutoPas/AutoPas/pull/745/commits/4a5aece0f8222be36761514e6c229ecbf982c33f). The functions in LJPotential can be used as constexpr. Unfortunately I could only use it in one test case as pure constexpr. But this one case shows also that it works as constexpr.